File: official/docs/csharp/carrier_accounts/RetrieveCarrierAccountList.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Xunit;
using Xunit.Sdk;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples;

public class Examples
{
    [Fact]
    public async Task RetrieveCarrierAccountList()
    {
        string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

        EasyPost.ClientManager.SetCurrent(apiKey);

        List<CarrierAccount> carrierAccounts = await CarrierAccount.All();

        new TestOutputHelper().WriteLine(JsonConvert.SerializeObject(carrierAccounts, Formatting.Indented));
    }
}

File: official/docs/java/carrier_accounts/RetrieveCarrierAccountList.java
package carrier_accounts;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.CarrierAccount;

import java.util.List;

public class RetrieveCarrierAccountList {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        List<CarrierAccount> carrierAccounts = CarrierAccount.all();

        System.out.println(carrierAccounts);
    }
}
